packages/react-charting/src/components/HorizontalBarChart/HorizontalBarChart.base.tsx

```diff
@@ -318,15 +318,4 @@ export class HorizontalBarChartBase extends React.Component<IHorizontalBarChartP
       isCalloutVisible: false,
     });
   };
-
-  private _getAccessibleDataObject = (accessibleData?: IAccessibilityProps, role: string = 'text') => {
-    accessibleData = accessibleData ?? {};
```

Comment:
Where is the culture value getting set as role in the original code?

Reply:
Here we are passing culture as the argument instead of text

![image](https://user-images.githubusercontent.com/103020020/183942336-0b7a63e7-e744-4b5b-9ff8-2dec74b26787.png)